lib/node_modules/@stdlib/blas/ext/base/dsnansumpw/docs/repl.txt

```diff
@@ -47,13 +47,14 @@
     -1.0
 
 
-{{alias}}.ndarray( N, x, stride, offset )
+{{alias}}.ndarray( N, x, strideX, offsetX )
     Computes the sum of single-precision floating-point strided array elements,
     ignoring `NaN` values and using pairwise summation with extended
```

Comment:
This looks like it can be phrased to be more similar to the description of the main export.